src/main/kotlin/org/jitsi/jicofo/jibri/JibriDetector.kt

```diff
@@ -130,6 +131,24 @@ class JibriDetector(
             this["available"] = getInstanceCount { it.status.isAvailable }
         }
 
+    val debugState: OrderedJsonObject
+        get() = OrderedJsonObject().also { debugState ->
+            debugState["is_sip"] = isSip
+            debugState["brewery_jid"] = breweryJid.toString()
+            instances.forEach { instance ->
+                val instanceJson = OrderedJsonObject().apply {
+                    this["health_status"] = instance.status.healthStatus?.status.toString()
+                    this["busy_status"] = instance.status.busyStatus?.status.toString()
+                    jibriInstances[instance.jid]?.let {
+                        this["reports_available"] = it.reportsAvailable
+                        this["last_failed"] = it.lastFailed.toString()
+                        this["last_selected"] = it.lastSelected.toString()
+                    }
+                }
+                debugState[instance.jid.resourceOrEmpty.toString()] = instanceJson
```

Comment:
Will this always be a resource, and not empty?

Reply:
I pushed a change that makes sure it is